argcomplete/finders.py

```diff
@@ -519,8 +519,14 @@ def quote_completions(
             special_chars = ""
             completions = [c.replace("'", r"'\''") for c in completions]
 
+        # PowerShell uses ` as escape character.
+        if os.getenv('PSModulePath'):
+            escape_char = '`'
+            special_chars = special_chars.replace('`', '')
```

Comment:
On Linux, why `\` is not in special chars?

Do I need to convert `` ` `` to some other string, and convert back to ``` `` ```?